include/halo/utils/cl_options.h

```diff
@@ -49,6 +49,14 @@ static llvm::cl::opt<bool> PrintAnalysisReport(
     "print-analysis-report", llvm::cl::desc("Print analysis report"),
     llvm::cl::init(false));
 
+static llvm::cl::opt<bool> ConvertToIpuGraphDef(
```

Comment:
These options are only for the "analyzer" tool. 
Shall we move them to "analyzer/driver.cc" ?

Reply:
done